dashboard-api/internal/server/middleware/organizations/organizations.go

```diff
@@ -0,0 +1,27 @@
+package organizations
+
+import (
+	"github.com/gin-gonic/gin"
+)
+
+type orgHeader struct {
+	orgID string `header:"VLAB_ORGANIZATION"`
```

Comment:
Cool, so the idea is that all requests should, in addition to a user, have org in the header? 

I'm not sure I love it. It feels a little non-traditional to me. It feels unclear if this is authorization or something else. 

I'm trying to work through the need. We have a couple of resources, right now: 

1. Studies. A study belongs to a single org.

I look at Vercel here, just because I've been using it recently, and notice that the projects are underneath teams in the URL: /{TEAM}/{PROJECT}. Which, in a resource/REST scenario, is easy to understand and makes sense to me. Studies belong to Orgs. 

2. Accounts / Connected Accounts. When one connects an external account, one inherently does so as an individual. But one connects the external account to... What? The Org? Or all Orgs the user is a part of? From an external perspective, it doesn't need to be the Org - once Facebook authorizes Vlab to act on behalf of an individual, Vlab has that permissionsfro period. Regardless of Org. From on internal perspective, I might connect my Facebook account to one Org, but not want to connect it to another Org. That seems reasonable to me. But then maybe there are two separate concepts: (a) did Nandan authorize Vlab to use Facebook or (b) did Nandan connect his Facebook to Org X. 

So one could either (1) subordinate account to an org or (2) separate those two concepts, subordinate (b) to the Org but leave (a) at an individual level. 

Thoughts?